index.js

```diff
@@ -27,6 +28,8 @@ io.of('/lobby').on('connect', socket => {
 
       results.push(vote);
 
+      user.vote = vote;
```

Comment:
Should probably clear it on reset as well, and set it to null instead of NaN.

Reply:
done